java8-assignment

manishmowall opened the pull request.

Commit:
Added src and .gitignore

Commit:
Added Lambda Expression Package

It contain four files

1> Apple model class file
2> AppleFilter functional interface file used for boolean test for filtering apple list
3> ApplePrinter functional interface used for printing Apple
detail as per condition
4> Application class containing main method and prettyPrintApples method

Commit:
Separated AppleFilter And ApplePrinter Interface

Created two package, one for ApplePrinter functional interface and
other for AppleFilter functional interface

Commit:
reformated code

Commit:
Added HEAVY_BENCHMARK Constant

added to lambdaexpression1/Application.java and lambdaexpression2/Application.java

murtaza-webonise commented on src/main/java/org/webonise/lambdaexpression1/Apple.java:
```diff
@@ -0,0 +1,26 @@
+   private final String color;
+   private final int id;
+
+   public Apple(int weight, String color, int id) {
```
make this arguments thread safe

murtaza-webonise commented on src/main/java/org/webonise/lambdaexpression1/Application.java:
```diff
@@ -0,0 +1,42 @@
+        Application application = new Application();
+        List<Apple> apples = application.generateApples();
+
+        ApplePrinter applePrinter1 = (apple) -> Integer.toString(apple.getWeight());
```
rename applePrinter1

murtaza-webonise commented on src/main/java/org/webonise/lambdaexpression1/Application.java:
```diff
@@ -0,0 +1,42 @@
+
+        return apples;
+    }
+}
```
add newline

Commit:
Made Changes To Lambda Expression

1> added new line at the end
2> added final to arguments of constructor
3> renamed functional interface variables

manishmowall commented:
made changes that were asked 
check it
Please comment if any corrections is required
